### PR TITLE
Calculate score

### PR DESCRIPTION
**Pull request by mathildegrimal**
#### Before submitting, make sure:

- [x] You have rebased your branch
- [x] You describe your changes
- [x] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

#### Describe below your changes:
added route and method to calculate score from questionnaire's responses

##### Describe below How to test:

When you implement the call to the route on FRONTEND, please make sure you send and object with :
```js
{
	user: "userId", //string or null
	responses:[{}] // array of objects with these properties : {questionId:string; answerId:string; impact:number}
}
```

You have to test from postman the route POST http://localhost:3001/ekh2o/questions/calculateScore with this body :
```json
{
	"user":"614d7133-b503-43db-be3e-26db874a48d4",
	"responses":[
		{
			"questionId": "15ecae5f-3b7c-405a-8d64-94bae51b702d",
    	"answerId": "a1b66597-3853-4b03-a410-6180c8613a44",
    	"impact": 3
		},
		{
			"questionId": "6ad4880f-1856-4ff1-8eb7-013f63853231",
    	"answerId": "d4aff741-89d0-4e08-9d55-d01401e8a719",
    	"impact": 150
		},
		{
			"questionId": "5518c7c7-daed-41c3-bb22-39aff6793407",
    	"answerId": "ca10b651-b6b2-4d5f-9576-92218e7f6d36",
    	"impact": 50
		},
		{
			"questionId": "2402e595-95e2-46c0-a68d-43d6afe96ffb",
    	"answerId": "481bfd1a-72c5-4c33-a674-20d7a1d63998",
    	"impact": 150
		},
		{
			"questionId": "9353c124-2c9e-4ed8-b06b-e76af65d0e57",
    	"answerId": "af90a4a1-e93a-4e09-845f-c32b9333d8b2",
    	"impact": 1
		},
		{
			"questionId": "03bbb2d8-31a1-4a73-b10b-2e3462ab6e2c",
    	"answerId": "ec606123-4a68-4362-a392-617da1947d0f",
    	"impact": 25
		},
		{
			"questionId": "b10ae0a6-1966-425a-979c-6a15bd101aa3",
    	"answerId": "7f1d9245-15df-4126-951c-e252568f8072",
    	"impact": 150
		},
		{
			"questionId": "56e21ea4-3fb1-42b1-a4d3-346f9eebc4ce",
    	"answerId": "9a8754da-8b89-45d0-a624-f8057e2feff8",
    	"impact": 2
		},
		{
			"questionId": "0359339d-c636-4884-9081-ebd443509dab",
    	"answerId": "754f0c25-920c-46d5-8a63-969c9bc2c24b",
    	"impact": 90
		},
		{
			"questionId": "dd6480fb-006f-49cf-a15d-23a667855ad5",
    	"answerId": "cc22a019-b174-46df-a2a3-9e235dd08bd3",
    	"impact": 1
		}
	]
}
```

### CODE REVIEW

File: src/Questions/questions.methods.ts [hunk 1]
```diff
@@ -7,3 +10,61 @@ export async function getAllQuestions() {
...
+    //post stat { user, score } -> return statId
+    const { data: statId } = await axios.post(
+      `${process.env.EKH2O_API_URL}stats`,
+      stat
+    );
+
+    const { id } = statId;
+
+    //modify reponses with statId property on every object and set statId property
+    for (const r of responses) {
```
Comment (Antonin-dev) [inline src/Questions/questions.methods.ts:58]: response plz

File: src/Questions/questions.methods.ts [hunk 2]
```diff
@@ -7,3 +10,61 @@ export async function getAllQuestions() {
...
+
+  //nombre de litre consommés en fonction du nombre de lessive par semaine * nb de semaine
+  score += responses[8].impact * 52.25;
+
+  //économie moyenne d'eau en cas d'installation de 3 mousseurs dans la maison
+  score *= responses[9].impact * 1 - 40 / 100;
+  console.log(user);
+  if (user) {
+    const stat = {
+      score: score,
```
Comment (Antonin-dev) [inline src/Questions/questions.methods.ts:45]: score,

File: src/Questions/questions.methods.ts [hunk 3]
```diff
@@ -7,3 +10,61 @@ export async function getAllQuestions() {
...
+
+  //vaisselle par jour * par quantité d'eau selon si lavage main ou machine
+  score += responses[6].impact * responses[7].impact * 7;
+
+  //nombre de litre consommés en fonction du nombre de lessive par semaine * nb de semaine
+  score += responses[8].impact * 52.25;
+
+  //économie moyenne d'eau en cas d'installation de 3 mousseurs dans la maison
+  score *= responses[9].impact * 1 - 40 / 100;
+  console.log(user);
```
Comment (Antonin-dev) [inline src/Questions/questions.methods.ts:42]: console.log

File: src/Questions/questions.methods.ts (hunk 1 above)
Comment (mathildegrimal) [reply src/Questions/questions.methods.ts:58]: ok

File: src/Questions/questions.methods.ts (hunk 2 above)
Comment (mathildegrimal) [reply src/Questions/questions.methods.ts:45]: ok

File: src/Questions/questions.methods.ts (hunk 3 above)
Comment (mathildegrimal) [reply src/Questions/questions.methods.ts:42]: ok

**Review by Antonin-dev — APPROVED** [review]

**Pull request merged**